File: orika-spring-boot-starter/src/main/java/net/rakugakibox/spring/boot/orika/OrikaMapperFactoryConfigurer.java
package net.rakugakibox.spring.boot.orika;

import ma.glasnost.orika.MapperFactory;

/**
 * The interface to configure {@link MapperFactory}.
 */
public interface OrikaMapperFactoryConfigurer {

    /**
     * Configures the {@link MapperFactory}.
     *
     * @param orikaMapperFactory the {@link MapperFactory}.
     */
    void configure(MapperFactory orikaMapperFactory);

}

File: orika-spring-boot-starter/src/main/kotlin/dev/akkinoc/spring/boot/orika/OrikaMapperFactoryConfigurer.kt
package dev.akkinoc.spring.boot.orika

import ma.glasnost.orika.MapperFactory

/**
 * The interface to configure [MapperFactory].
 */
interface OrikaMapperFactoryConfigurer {

    /**
     * Configures the [MapperFactory].
     *
     * @param orikaMapperFactory The [MapperFactory].
     */
    fun configure(orikaMapperFactory: MapperFactory)

}
